src/subcommand/wallet/mint.rs

```diff
@@ -6,6 +6,11 @@ pub(crate) struct Mint {
   fee_rate: FeeRate,
   #[clap(long, help = "Mint <RUNE>. May contain `.` or `•`as spacers.")]
   rune: SpacedRune,
+  #[clap(
+    long,
+    help = "Amount of postage to include in the inscription. Default `10000sat`."
```

Comment:
This postage is for the mint of a rune, so no inscription inside. 

```suggestion
    help = "Amount of postage to include in the minted rune output. Default `10000sat`."
```